celery/app/trace.py

```diff
@@ -623,3 +622,7 @@ def __protected_call__(self, *args, **kwargs):
             return orig(self, *args, **kwargs)
         BaseTask.__call__ = __protected_call__
         BaseTask._stackprotected = True
+
+
+def _info(fmt, context):
```

Comment:
Thank you for the suggestion. I've added a docstring including the reason for adding extra.